src/vtbackend/Terminal.cpp

```diff
@@ -1823,17 +1823,6 @@ void Terminal::hardReset()
     _state.inputGenerator.reset();
 }
 
-void Terminal::forceRedraw(std::function<void()> const& artificialSleep)
```

Comment:
I would probably keep this function. It may serve good also as its own action :)